### PR TITLE
Add group chrome kafka message

### PR DESCRIPTION
**Pull request by lpichler**
We need to send kafka message when 
group is created/updated/deleted.

format of message is cloud event, example of sent message to `platform.chrome` topic:

```
{
  "specversion": "1.0.2",
  "type": "data.invalidation",
  "source": "/rbac/v1/status/",
  "id": "480e9e77-f71b-41ee-8d21-11c117afc6d9",
  "time": "2023-07-04T13:05:06.345064+00:00",
  "datacontenttype": "application/json",
  "data": {
    "broadcast": false,
    "organizations": [
      "11111"
    ],
    "payload": {
      "entityType": "rbac.group",
      "entityId": "be5242af-eabe-4925-b176-b45cc251d7df",
      "eventType": "create"
    }
  }
}
```

This is experimental effort - tests are not included for now.

### CODE REVIEW

**Comment by lpichler** [comment]
not this is ready for review @abaiken @Hyperkid123

**Review by Hyperkid123 — COMMENTED** [review]
The sample message passes the Chrome service validation. so LGTM :+1:

File: rbac/core/kafka.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class RBACProducer:
     def get_producer(self):
         """Init method to return fake kafka when flag is set to false."""
         if not hasattr(self, "producer"):
-            if settings.DEVELOPMENT or settings.MOCK_KAFKA or not settings.KAFKA_ENABLED:
+            if settings.MOCK_KAFKA or not settings.KAFKA_ENABLED:
```
Comment (abaiken) [inline rbac/core/kafka.py:38]: Do we not want to fake kafka if Development is true?

Comment (lpichler) [reply rbac/core/kafka.py]: Oh good eye, I forgot to remove - it was just for my debugging purposes. Thanks!

**Review by abaiken — APPROVED** [review]
LGTM 🎉

**Comment by lpichler** [comment]
https://issues.redhat.com/browse/RHCLOUD-26631

**Pull request merged**